openwisp_notifications/handlers.py

```diff
@@ -158,13 +157,23 @@ def send_email_notification(sender, instance, created, **kwargs):
     instance.save()
 
 
+@receiver(post_save, dispatch_uid='notification_related_object_cache')
+@receiver(post_delete, dispatch_uid='notification_related_object_cache_deleted')
+def invalidate_notification_related_object_cache(sender, instance, **kwargs):
+    if kwargs.get('created', False):
+        return
+    instance_id = getattr(instance, 'pk', None)
+    if instance_id:
+        Notification.invalidate_related_object_cache(instance_id)
```

Comment:
here you're invalidating based on instance_id but not supplying any info on the content type which doesn't look right